src/app/chat/layout.tsx

```diff
@@ -8,13 +7,9 @@ export default function ChatLayout({
   return (
     <SidebarProvider>
       <ChatSidebar />
-      <div className="flex h-full min-h-dvh w-full flex-col bg-chat-background text-white">
-        <ClientOnly>
-          <SidebarTrigger className="border-0" />
-          <div className="flex h-screen w-full flex-col bg-chat-background text-white">
-            {children}
-          </div>
-        </ClientOnly>
+      <div className="flex h-screen w-full flex-col bg-chat-background text-white">
```

Comment:
**issue:** Don't use fixed height values except for images. As explained [in my discord message](https://discord.com/channels/687360174377533442/1306222236499443724/1319665574879563786), in this case, this approach blocks you from implementing proper chat page scrolling
```suggestion
      <div className="flex min-h-screen w-full flex-col bg-chat-background text-white">
```
**note:** You might remember my first bigger PR when i mentioned that `w-full` is not required on `display: block` elements, which is the default setting, but since we're using the sidebar this is required here (just wanted to point this out)

Reply:
I admire your knowledge on these all nitty-gritty css stuff
For me assembly is easier than css 😭